### DISCUSSION TITLE
Not getting Galaxy Brain Badge with >= 2 discussions answered

### DISCUSSION THREAD

**Discussion by port19x** [community/community#43854] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Bug

### Body

Hi, I'm not getting the galaxy brain badge even tho I have been selected as the answer to multiple discussions.
I did not select myself as the answer for these following two:
- [ani-cli 963](https://github.com/pystardust/ani-cli/discussions/963)
- [ani-cli 957](https://github.com/pystardust/ani-cli/discussions/957)

If it's relevant, I have commented and selected myself as the answer for several other discussions

Can now confirm the workaround is working. Thx @linuxlawson

**Comment by hughbreslin** [comment]
I am also seeing this

**Comment by aliencaocao** [comment]
same here

**Comment by radar-blip** [comment]
@port19x You may need to go back and unmark the answers where you selected yourself (if possible).
Also, you might need to unmark another answer so the count starts over.

See selected answer here: #18454

**Comment by Fasping** [comment]
Same here 🥺

**Comment by github-staff** [comment, marked as answer]
Because of an influx of users creating low quality, nonsensical or spam posts in the Community with the aim of earning certain Achievements, we’ve made the difficult decision to disable the ability to earn Achievements in our Community.

Note that this change does not affect your ability to earn badges in other projects and communities across GitHub. This change only applies to GitHub’s Community Discussions.

You can learn more about this decision in our announcement post here [Achievements will no longer be available in the Community](https://github.com/orgs/community/discussions/106536).

Let us know if you have any other questions 🚀
